### ISSUE TITLE
Pushing to PyPI for RC2

### ISSUE THREAD

**Issue by springcoil** [pymc-devs/pymc#1419]
Hi, who is pushing it to PyPI for RC2? Is it you Chris?

**Comment by fonnesbeck** [comment]
I will tag the release and post to PyPI

**Comment by springcoil** [comment]
Cool :)

**Comment by twiecki** [comment]
@fonnesbeck I think we can move ahead with this now.

**Comment by ericmjl** [comment]
Is #1339 resolved in RC2? I'm happy to start making a conda-forge recipe for this if it's not already being done.

**Comment by springcoil** [comment]
I think it's ok to move forward with a conda-forge recipe. Any objections @twiecki @fonnesbeck

**Comment by twiecki** [comment]
@ericmjl should be resolved, but we still need the release on pypi.

**Comment by ericmjl** [comment]
@twiecki: no problem with waiting for the release on PyPI. I'll get the recipe prepped nonetheless.

**Comment by fonnesbeck** [comment]
[completed](https://pypi.python.org/pypi/pymc3)

**Comment by twiecki** [comment]
@ericmjl Do you think getting rc2 on conda forge is going to be quick? If so, we might as well wait with the announcement until that's ready.

**Comment by ericmjl** [comment]
@twiecki: I'll have to see how long the builds take, as well as when the conda-forge team approves the PR. What's your personal cut-off time? This evening, or next 1-2 days, or the end of the week?

**Comment by fonnesbeck** [comment]
@ericmjl There is no cutoff _per se_. As much as everyone wants to see the release, there is no reason to rush it.

**Comment by twiecki** [comment]
For reference: https://github.com/conda-forge/staged-recipes/pull/1704

**Comment by ericmjl** [comment]
@fonnesbeck thanks! :smile:

P.S. ever since attending your tutorial @ SciPy 2014 + listening to @twiecki's talk @ Boston Bayesians, I've started to have a ton of fun w/pymc3. Lovin' it!

**Comment by ericmjl** [comment]
It's looking like there's a bunch of dependencies that need to be built via conda-forge first. I'm slowly running into them:
- nbsphinx
- commonmark
- recommonmark

And then dependencies of the dependencies:
- configparser

I'll keep you guys posted.

**Comment by twiecki** [comment]
Hm, those are just required to build the docs, perhaps we can make them optional in pymc3?

**Comment by ericmjl** [comment]
Perhaps so, I guess? I think I can modify the pymc3 build script so that they aren't included or required.

**Comment by twiecki** [comment]
great

**Comment by ericmjl** [comment]
It's in building right now: https://github.com/conda-forge/staged-recipes/pull/1705

**Comment by ericmjl** [comment]
Not quite sure why this error is happening on the CI-build: 

https://circleci.com/gh/conda-forge/staged-recipes/9809?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Any ideas, @twiecki?

**Comment by twiecki** [comment]
`creating /opt/conda/envs/_build_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.5/site-packages/pymc3`

Is that weird directory name at fault perhaps?

**Comment by ericmjl** [comment]
Maybe, though on my local build that weird dir also shows up. I'll see what the conda-forge folks say about it as well.

**Comment by ericmjl** [comment]
The PR has been merged! :smile: https://github.com/conda-forge/staged-recipes/pull/1705

Just waiting for the feedstock to be created, and it'll be up on `conda-forge`. There is one minor issue with the Windows build - had to do with matplotlib. The `matplotlib` folks will have to take care of it.

I also submitted PRs for the dependencies that are needed in building the docs, just to make things complete.
